src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -128,6 +152,39 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
         return Promise.resolve(this._store.getState());
     }
 
+    private async _processAdditionalAction(error: any): Promise<InternalCheckoutSelectors | never> {
```

Comment:
we usually try to avoid using `any` in this repo, could you please type the error param?